jgiwinski/My-Name-Is-Dad
# Fix/refactor

## What does this PR do?

- Removes propsSetToState from SearchForm.js
- Changes conditional on header buttons so that they don't move to a different spot on click
- Change .catch error handling to use alerts
- Delete some unused code

## How should it be tested in the UI?

- Click on search / humble abode buttons and see that you no longer have to move your cursor to click them.
